### PR TITLE
Handle python projects that have a manage.py file

### PR DESCRIPTION
**Pull request by EnoshAnwar**
Django and flask projects usually would have this file and use it to run tests.

Decided to create a new kind of file label since in this case we were going to be
using a single file to determine which test command to run.

### CODE REVIEW

**Review by skaasten — APPROVED** [review]

File: cmd/inferconfig/testdata/expected/circleci-demo-python-django.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # This config was automatically generated from your source code
-# Stacks detected: deps:python:.,package_manager:pipenv:.
+# Stacks detected: deps:python:.,file:manage.py:.,package_manager:pipenv:.
```
Comment (polymeris) [inline cmd/inferconfig/testdata/expected/circleci-demo-python-django.yml:2]: Maybe I'd call this e.g "framework:django" instead of "manage.py"

Comment (EnoshAnwar) [reply cmd/inferconfig/testdata/expected/circleci-demo-python-django.yml:2]: That is what I had before 😅 But then saw flask projects also containing that file https://github.com/CircleCI-Public/circleci-demo-python-flask. Can make the change if we're fine with labelling other frameworks that have manage.py django

**Review by polymeris — APPROVED** [review]

File: cmd/inferconfig/testdata/expected/circleci-demo-python-django.yml (hunk 1 above)
Comment (polymeris) [reply cmd/inferconfig/testdata/expected/circleci-demo-python-django.yml:2]: Aha! `file:manage.py` is fine then, I'll leave the decision to you.

File: generation/internal/python.go [hunk 2]
```diff
@@ -7,15 +7,27 @@ import (
...
+	if hasManagePy {
+		steps = append(steps, config.Step{
+			Name:    "Run tests",
+			Type:    config.Run,
+			Command: "python manage.py test",
+		})
+		return steps
+	}
+
+	steps = append(steps, []config.Step{
```
[the remark this answers is not in the document]
Comment (EnoshAnwar) [reply generation/internal/python.go:30]: Yeah, I think it's because we are not installing the dev dependencies. Will take a look 👍

**Pull request merged**